_rules/meta-viewport-b4f0c3.md

```diff
@@ -263,3 +266,5 @@ This viewport `meta` element does not have a `content` attribute.
 [page]: https://www.w3.org/TR/WCAG21/#dfn-web-page-s 'Web page'
 [user-scalable]: https://www.w3.org/TR/css-device-adapt-1/#user-scalable 'The user-scalable property'
 [visible]: #visible 'Definition of visible'
+
+[css pixels]: https://www.w3.org/TR/WCAG21/#dfn-css-pixels 'WCAG 2.1 definition, CSS pixels
```

Comment:
Shouldn't that link directly to https://www.w3.org/TR/css3-values/#reference-pixel instead?

Reply:
Might as well, I guess